tests/src/test/java/ca/bc/gov/open/jag/efiling/stepDefinitions/getSubmissionSheetSD.java

```diff
@@ -0,0 +1,59 @@
+package ca.bc.gov.open.jag.efiling.stepDefinitions;
+
+import ca.bc.gov.open.jag.efiling.models.UserIdentity;
+import ca.bc.gov.open.jag.efiling.services.FilingPackageService;
+import ca.bc.gov.open.jag.efiling.services.OauthService;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.restassured.path.json.JsonPath;
+import io.restassured.response.Response;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class getSubmissionSheetSD {
+
+    private OauthService oauthService;
+    private UserIdentity actualUserIdentity;
+    private FilingPackageService filingPackageService;
+
+    private Response actualSubmissionSheetResponse;
+
+    private Logger logger = LoggerFactory.getLogger(getSubmissionSheetSD.class);
+
+    public getSubmissionSheetSD(OauthService oauthService, FilingPackageService filingPackageService) {
+        this.oauthService = oauthService;
+        this.filingPackageService = filingPackageService;
+    }
+
+    @Given("user account is authenticated")
+    public void userAccountIsAuthenticated() {
+
+        actualUserIdentity = oauthService.getUserIdentity();
+    }
+
+    @When("user submits request to get submission sheet")
+    public void getSubmissionSheetRequest() {
+        logger.info("Requesting submission sheet from filing packages");

```

Comment:
You may want to flip these log statements to `.debug` or `.trace`, leaving `.info` for high-level course-grained information (ie. app started up with these parameters).

Reply:
Good point! I wasn't sure if .debug will be too much information to parse through or will be take a bit extra time on the CI test runs when implemented for all requests. But a worth trying... Thanks @KDNeufeld!

Reply:
I don't see too much issue having the integration test verbose, something we could consider, but I think we always wants to see those logs in the output, what do you guys think ?